drivers/include/motor_driver.h

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright (C) 2018 Gilles DOFFE <g.doffe@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_motor DC Motor Driver
+ * @ingroup     drivers_actuators
+ * @brief       High-level driver for DC motors
+ *
+ * This API aims to handle DC motor analogic driver.
```

Comment:
Most of H-brdige applications is for driving motors and brake capability is mainly dedicated to motors.
Moreover we should be able to drive brushless motor drivers like this one soon:
https://cogip.duckdns.org/news/moteurs-brushless
So yes it could mainly be used for hbridge but not only.

Reply:
When I read brushless I thought you were referring so a some "smart" brushess that allows driving with a simple PWM signal like it was a regular DC motor.

That link refers to a brushless dc (BLDC) driver. Driving that stuff its way more complicated. You need a three phase inverter (3 half-bridges) and you need to sense the back-emf (either via an ADC or by using comparators) and, above all, some pretty tight timing requirements.